src/modules/display.js

```diff
@@ -1,7 +1,7 @@
 const display = (game) => {
   const toli = document.getElementsByTagName('ul')[0];
   const gameElement = document.createElement('li');
-  gameElement.innerHTML = `<li>${game.user}: ${game.score}</li>`;
+  gameElement.innerHTML = `<li class=score-li >${game.user}: ${game.score}</li>`;
   toli.appendChild(gameElement);
```

Comment:
- [ ] [Optional] When you update the leaderboard the scores render twice, so kindly when you update the scores you suppose to replace the old scores with the new updated ones.